docker-compose/monitor/jaeger-v2-config.yml

```diff
@@ -0,0 +1,47 @@
+service:
+  extensions: [jaeger_storage, jaeger_query]
+  pipelines:
+    traces:
+      receivers: [otlp, jaeger, zipkin]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter, spanmetrics]
```

Comment:
is `spanmetrics` considered an exporter? I don't quite know how connectors work in otelcol.